controllers/controller.js

```diff
@@ -7,8 +7,18 @@ exports.getCategories = (req, res) => {
   });
 };
 
-exports.getEndpoints = (req, res) => {
+exports.getEndpoints = (req, res, next) => {
   return fs.readFile("endpoints.json", "utf-8").then((result) => {
     res.status(200).send({ endpoint: result });
   });
 };
+
+exports.getReviewId = (req, res, next) => {
+  let { review_id } = req.params;
+  fetchReviewId(review_id)
+    .then((data) => {
+      console.log(data);
+      res.status(200).send({ review: data });
+    })
+    .catch((err) => next(err));
```

Comment:
Good work adding this .catch block to deal with any errors passed to it (for example, the promise rejection in your model will pass that to this .catch block)